astropy/coordinates/builtin_frames/heliocentric.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import (absolute_import, unicode_literals, division,
+                        print_function)
+
+from ..representation import SphericalRepresentation
+from ..baseframe import (BaseCoordinateFrame, RepresentationMapping,
+                         TimeFrameAttribute)
+from .utils import DEFAULT_OBSTIME
+
+class Heliocentric(BaseCoordinateFrame):
```

Comment:
oh, this is an interesting point. In SunPy we have a frame named `Heliocentric` I think all manner of things could explode if there are two frames on the same graph with the same name? @eteq ??  Perhaps both need to be more specific in their naming?